privacyidea/static/components/token/controllers/tokenDetailController.js

```diff
@@ -117,6 +117,24 @@ myApp.controller("tokenDetailController", function ($scope,
             $rootScope.previousState.params);
     };
 
+    $scope.unassignAskAdmin = function() {
+        $scope.confirm(
+            $scope.confirm_action_levels["easy"],
+            gettextCatalog.getString("Unassign Token"),
+            gettextCatalog.getString("Do you really want to remove this token from this users account?"),
```

Comment:
I think this is a type -> "user's"

I would simply do: "Do you really want to unassign this token from the user"